lib/ansible/modules/system/sefcontext.py

```diff
@@ -27,16 +27,15 @@
     aliases: [ path ]
   ftype:
     description:
-    - File type.
-    - The following file type options can be passed;
-      C(a) for all files,
-      C(b) for block devices,
-      C(c) for character devices,
-      C(d) for directories,
-      C(f) for regular files,
-      C(l) for symbolic links,
-      C(p) for named pipes,
-      C(s) for socket files.
+    - "The following file type options are available:"
```

Comment:
We tend to use the first line in the description to explain what the option is for.
Granted "File type." was not really helpful, but a short sentence would be nice.

Rationale: In the future we want to highlight the first line from the description in the module docs.

Reply:
That makes sense to me. I'll go tinker with that a bit more and put up a new revision.